es_jsonschema/mapping.py

```diff
@@ -80,47 +129,60 @@ def get_es_type(self, json_type, json_format=None):
 
     @property
     def date_format(self):
-        return getattr(self, '__date_format', None)
+        """Return the date format used in elasticsearch mappings."""
+        return self.__date_format
```

Comment:
Avoid using Java style code with setters and getters unless you are doing something special in your getter/setter. Enjoy Python world!